graviti/utility/file.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022 Graviti. Licensed under MIT License.
+#
+
+"""Graviti file related class."""
+
+from functools import partial
+
+from tensorbay.utility.file import URL, RemoteFileMixin
+
+_URL = partial(URL, updater=lambda: "update is not supported currently")
+
+
+class File(RemoteFileMixin):
+    """This class represents the file on Graviti platform.
+
+    Arguments:
+        url: The url string of the file.
+        checksum: The checksum of the file.
+
+    """
+
+    def __init__(self, url: str, checksum: str) -> None:
+        super().__init__("")
+        self.url = _URL(url)
+        self._checksum = checksum
+
+    def _repr_head(self) -> str:
+        return f'{self.__class__.__name__}("{self._checksum}")'
```

Comment:
will the full checksum too long？will it be better if we just display the first 7~8 letters of the checksum?

Reply:
@AChenQ

Reply:
We can just display the first 8 letters of the checksum now.